packages/itwinui-css/src/button/borderless.scss

```diff
@@ -21,16 +19,16 @@
   }
 
   &:where([disabled], :disabled, [aria-disabled='true']) {
-    --_iui-button-text-color: var(--iui-text-color-placeholder);
+    --_iui-button-text-color: var(--iui-color-foreground-5);
     --_iui-button-background-color: transparent;
     --_iui-button-border-color: transparent;
   }
 }
 
 @mixin iui-button-borderless-active {
-  --_iui-button-background-color: rgba(var(--iui-color-foreground-primary-rgb), var(--iui-opacity-6));
+  --_iui-button-background-color: var(--iui-color-background-subtle-primary);
```

Comment:
This color is barely visible on darker background e.g. inside info-panel 
![image](https://user-images.githubusercontent.com/36186912/185614385-0fb7012e-adb9-4854-8eb2-d5619d9687f9.png)

Reply:
Should be fixed now